downstream/modules/aap-hardening/con-automation-use-secrets.adoc

```diff
@@ -15,7 +15,7 @@
 
 You can grant users and teams the ability to use these credentials without actually exposing the credential to the user. This means that if a user moves to a different team or leaves the organization, you don’t have to re-key all of your systems.
 
-{ControllerName} uses SSH (or the Windows equivalent) to connect to remote hosts . To pass the key from the {ControllerName} to SSH, the key must be decrypted before it can be written to a named pipe. {ControllerNameStart} then uses that pipe to send the key to SSH (so that it is never written to disk). If passwords are used, the {ControllerName} handles those by responding directly to the password prompt and decrypting the password before writing it to the prompt.
+{ControllerNameStart} uses SSH (or the Windows equivalent) to connect to remote hosts . To pass the key from the {ControllerName} to SSH, the key must be decrypted before it can be written to a named pipe. {ControllerNameStart} then uses that pipe to send the key to SSH (so that it is never written to disk). If passwords are used, the {ControllerName} handles those by responding directly to the password prompt and decrypting the password before writing it to the prompt.
```

Comment:
Remove extra space at the end of the first sentence: "{ControllerNameStart} uses SSH (or the Windows equivalent) to connect to remote hosts ."